unity/Assets/Scripts/ArmAgentController.cs

```diff
@@ -20,6 +20,28 @@ public class ArmAgentController : PhysicsRemoteFPSAgentController {
             return arm;
         }
 
+        public void MoveArmRelative(
```

Comment:
I'd like a brief header comment here as well as for each of the new user accessible actions in this PR. Details like what each parameter does, how it is different from the default `MoveArm` action would streamline updating the documentation.

Reply:
Added in #801.